cards.html

```diff
@@ -4,9 +4,121 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/0a707a45d9.js" crossorigin="anonymous"></script>
+
+    <title>Cards</title>
 </head>
 <body>
-    
+    <!-- CARD CONTAINER-1 -->
+    <section class="card-container">
+        <div class="content">
+        <img class="image" src="assets/shirt.jpg" alt="img"/>
+        <div class="heading">
+            <h4>
+            <strong>Puma</strong>
+            <span class="btn btn-icon-primary wishlist-btn"><i class="far fa-heart"></i></span>
+            </h4>
+            <h5>White men regular cotton t-shirt.</h5>
+        </div>
+        <h4 class="item-cost"><strong><i class="fas fa-rupee-sign"></i> 1,899</strong></h4>
+        
+        </div>
+        <div>
+        <a><button class="btn btn-solid-primary">Buy Now</button></a>
+        <a class="btn btn-outline-primary"><span><i class="fas fa-shopping-cart"></i></span></a>
+        </div>
+    </section>
+    <!-- CARD CONTAINER-2 -->
+    <section class="card-container">
+        <div class="content">
+            <div class="image">
+                <img src="assets/shirt.jpg" alt="img"/><span class="status-badge badge-icon status-badge-number">NEW</span>
+            </div>
+        <div class="heading">
+            <h4><strong>Puma</strong>
+            <span class="btn btn-icon-primary wishlist-btn"><i class="far fa-heart"></i></span>
+            </h4>
+            <h5>White men regular cotton t-shirt.</h5>
+        </div>
+        <h4 class="item-cost"><strong><i class="fas fa-rupee-sign"></i> 1,899</strong></h4>
+        <div>
+        <a><button class="btn btn-solid-primary">Buy Now</button></a>
```

Comment:
You can name the solid primary button as btn-primary-solid
Similarly for outline primary button: btn-primary-outline.
One of several reasons for this naming convention is that it helps when you are coding your markup. In that case, if you start writing btn-primary, your VS Code can suggest you both variations for btn-primary, i.e, the solid one and the outline one

Reply:
Actually i have given the same class name in buttons component so that i would not have to style it again differently... will keep this in mind when i refactor my code after completion. Otherwise i might get confused.